Release exclusive_focus when a layer surface drops exclusive keyboard focus.
Drop after dwl 0.9.

diff --git a/dwl.c b/dwl.c
index e571b45..621c1af 100644
--- a/dwl.c
+++ b/dwl.c
@@ -972,10 +972,15 @@ commitlayersurfacenotify(struct wl_listener *listener, void *data)
 		return;
 	}
 
-	if (layer_surface == exclusive_focus
+	if (l == exclusive_focus
 			&& layer_surface->current.keyboard_interactive !=
-				ZWLR_LAYER_SURFACE_V1_KEYBOARD_INTERACTIVITY_EXCLUSIVE)
+				ZWLR_LAYER_SURFACE_V1_KEYBOARD_INTERACTIVITY_EXCLUSIVE) {
 		exclusive_focus = NULL;
+		if (layer_surface->current.keyboard_interactive ==
+					ZWLR_LAYER_SURFACE_V1_KEYBOARD_INTERACTIVITY_NONE
+				&& layer_surface->surface == seat->keyboard_state.focused_surface)
+			focusclient(focustop(selmon), 1);
+	}
 
 	if (layer_surface->current.committed == 0 && l->mapped == layer_surface->surface->mapped)
 		return;
